internal/lint/lint.go

```diff
@@ -25,10 +25,10 @@ type Rule interface {
 }
 
 // New creates a new Lint.
-func New(rules []Rule, rootPath string, excludesDirs []string) *Lint {
+func New(rules []Rule, rootPath string, ignoreDirs []string) *Lint {
 	return &Lint{
-		rules:        rules,
-		rootPath:     rootPath,
-		excludesDirs: excludesDirs,
+		rules:      rules,
+		rootPath:   rootPath,
+		ignoreDirs: ignoreDirs,
```

Comment:
Why `ingore` instead of `exclude`?
`ingore` key work used in buf?

Reply:
Yeah, only because of buf